api/app/tests/apps/api/v1/test_private_signal_endpoint.py

```diff
@@ -257,6 +259,38 @@ def test_create_with_status(self):
 
         self.assertEqual(400, response.status_code)
 
+    @patch("signals.apps.api.v1.validation.address.base.BaseAddressValidation.validate_address",
+           side_effect=AddressValidationUnavailableException)  # Skip address validation
+    def test_create_initial_without_stadsdeel(self, validate_address):
+        # Create initial Signal, check that it reached the database.
+        signal_count = Signal.objects.count()
+
+        create_initial_data = self.create_initial_data
+        del(create_initial_data['location']['stadsdeel'])
+        del(create_initial_data['location']['address'])
+        del(create_initial_data['location']['buurt_code'])
+
+        y, x = create_initial_data['location']['geometrie']['coordinates']
+        geometry = MultiPolygon([Polygon.from_bbox([x - 100, y - 100, x + 100, y + 100])])
```

Comment:
Coordinates in WGS84 are longitude in [-180, 180] and latitude in [-90, +90] so doing +- 100 can create invalid coordinates. This would be save in the Rijksdriehoek systeem (EPSG:28992) that uses meters.

Reply:
Did not knew that. Changed it so that it is correct

Reply:
I added a box around the centrum of Amsterdam, so that I know that the coordinates I am using to create a Signal are correct.